Library/Core/UnoCore/Targets/Android/app/build.gradle

```diff
@@ -41,9 +34,6 @@ repositories {
         }
     }
 #endif
-#if @(Gradle.Model.Repository:IsRequired)
-    @(Gradle.Model.Repository:Join('\n'))
```

Comment:
@cbaggers added this in 0cb3d6ed52ffc19624877959038a24b0e012aa5a from the old history. Perhaps he can explain why?

Reply:
I've added a fix for potential `Gradle.Model.Repository` users in de594a3089b337d528420fcb2e7dfed64236c10b.